clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -35,10 +35,10 @@ class AddressSpace<string _AS> {
 // the one it will be lowered to.
 // This file assumes it will get lowered to generic or private.
 def DefaultAS    : AddressSpace<"clang::LangAS::Default">;
-def PrivateAS    : AddressSpace<"clang::LangAS::opencl_private">;
-def GlobalAS     : AddressSpace<"clang::LangAS::opencl_global">;
+def PrivateAS    : AddressSpace<"clang::LangAS::sycl_private">;
```

Comment:
to answer your question @bader, this was meant to be used in both OpenCL C and SYCL. However, I think your change has no effect as this line https://github.com/intel/llvm/pull/3634/files#diff-e0ad0cd86168ae6b1bdbc14d96dce7bd4de45ec1da69b9ac18dacb7adf1e453cR6520 is doing the conversion based on the language opts.